src/renderer/components/layout/sidebar.tsx

```diff
@@ -7,99 +7,34 @@ import styles from "./sidebar.module.scss";
 import type { TabLayoutRoute } from "./tab-layout";
 
 import React from "react";
-import { disposeOnUnmount, observer } from "mobx-react";
-import { cssNames, Disposer } from "../../utils";
-import { Icon } from "../icon";
-import { Workloads } from "../+workloads";
-import { UserManagement } from "../+user-management";
-import { Storage } from "../+storage";
-import { Network } from "../+network";
-import { crdStore } from "../+custom-resources/crd.store";
-import { CustomResources } from "../+custom-resources/custom-resources";
+import { observer } from "mobx-react";
+import { cssNames } from "../../utils";
 import { isActiveRoute } from "../../navigation";
-import { isAllowedResource } from "../../../common/utils/allowed-resource";
-import { Spinner } from "../spinner";
 import { ClusterPageMenuRegistration, ClusterPageMenuRegistry, ClusterPageRegistry, getExtensionPageUrl } from "../../../extensions/registries";
 import { SidebarItem } from "./sidebar-item";
-import { Apps } from "../+apps";
-import * as routes from "../../../common/routes";
-import { Config } from "../+config";
 import { catalogEntityRegistry } from "../../api/catalog-entity-registry";
 import { SidebarCluster } from "./sidebar-cluster";
-import type { KubeObjectStore } from "../../../common/k8s-api/kube-object.store";
-import type { KubeObject } from "../../../common/k8s-api/kube-object";
-import { withInjectables } from "@ogre-tools/injectable-react";
-import kubeWatchApiInjectable
-  from "../../kube-watch-api/kube-watch-api.injectable";
-
-interface Props {
+import { TabRoutesSidebarItems } from "./tab-routes-sidebar-items";
+import { ConfigSidebarItem } from "../+config/sidebar-item";
+import { ClusterSidebarItem } from "../+cluster/sidebar-item";
+import { NodesSidebarItem } from "../+nodes/sidebar-item";
+import { WorkloadsSidebarItem } from "../+workloads/sidebar-item";
+import { NetworkSidebarItem } from "../+network/sidebar-item";
+import { StorageSidebarItem } from "../+storage/sidebar-item";
+import { NamespacesSidebarItem } from "../+namespaces/sidebar-item";
+import { EventsSidebarItem } from "../+events/sidebar-item";
+import { HelmSidebarItem } from "../+helm/sidebar-item";
+import { UserManagementSidebarItem } from "../+user-management/sidebar-item";
+import { CustomResourcesSidebarItem } from "../+custom-resources/sidebar-item";
+
+interface SidebarProps {
   className?: string;
 }
 
-interface Dependencies {
-  subscribeStores: (stores: KubeObjectStore<KubeObject>[]) => Disposer
-}
-
 @observer
-class NonInjectedSidebar extends React.Component<Props & Dependencies> {
+export class Sidebar extends React.Component<SidebarProps> {
```

Comment:
Love this. Next step is to get rid of Open Closed Principle -break in `getTabLayoutRoutes`. But good enough for now.